spec/helpers/your_details_form_helper_spec.rb

```diff
@@ -0,0 +1,44 @@
+RSpec.describe Wpcc::YourDetailsFormHelper, type: :helper do
+  describe '.disable_minimum_contribution_option?' do
+    subject { helper.disable_minimum_contribution_option? }
+
+    context 'salary lower than threshold' do
+      let(:form) do
+        Wpcc::YourDetailsForm.new(age: age,
+                                  gender: gender,
+                                  salary: salary,
+                                  salary_frequency: salary_frequency,
+                                  contribution_preference: pref)
+      end
+
+      before do
+        form.valid?
+        assign(:your_details_form, form)
+      end
+
+      context 'make minimum contribution preference' do
+        let(:age)    { 35        }
```

Comment:
This is a discussion to be had about our code standard and what we are going to use...
I personally am not a fan of the trailing `}` to be in line. I like having the opening `{` to be in line, but then believe there should be only one space between the braces and the contents. i.e. `{ 35 }`
Happy to go with the majority vote though...
@Tr4pSt3R @margOnline @tomas-stefano